src/impression.js

```diff
@@ -233,11 +233,13 @@ function applyResponse(win, response) {
       return;
     }
 
+    const viewer = Services.viewerForDoc(win.document);
     const currentHref = win.location.href;
     const url = parseUrl(adLocation);
     const params = parseQueryString(url.search);
     const newHref = addParamsToUrl(currentHref, params);
     win.history.replaceState(null, '', newHref);
+    viewer.maybeUpdateFragmentForCct();
```

Comment:
Doesn't meant to overwrite fragment here. 
Maybe we should fix the issue for all, instead of just for Cct. Thanks.

Reply:
IMO this is existing behavior and shouldn't need to be tied to this PR. A TODO should suffice. :)